modules/con_example-yaml-file-changes.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_example-yaml-file-changes_{context}"]
+= Example YAML file changes
+
+You can dynamically add a code snippet in the YAML editor for customizing a developer catalog.
+
+Use the following snippet to display all the sub-catalogs by setting the _state_ type as *Enabled*.
+[source,yaml]
+----
+spec:
+  customization:
+    developerCatalog:
+      categories:
+      types:
+        state: Enabled
+----
+
+Use the following snippet to disable all sub-catalogs by setting the _state_ type as *Disabled*:
```

Comment:
Be consistent with the use of either . or : at the end of these three "Use the following..." statements

Reply:
Fixed.

Reply:
I don't see this change?